services/ui-src/src/types/index.ts

```diff
@@ -152,9 +152,13 @@ export interface ReportShape extends ReportKeys {
   submittedBy?: string;
   submitterEmail?: string;
   submittedOnDate?: number;
+  formTemplate?: ReportJson;
+  fieldData?: AnyObject;
+  archived?: boolean;
+}
+export interface ReportShape extends DashboardReportShape {
```

Comment:
Nit: Add the space here back in.
```suggestion
}

export interface ReportShape extends DashboardReportShape {
```